app/controllers/providers/omniauth_callbacks_controller.rb

```diff
@@ -34,8 +34,10 @@ def auth_data
       request.env['omniauth.auth']
     end
 
+    # :nocov:
     def office_codes
-      auth_data.info.office_codes
+      auth_data.info&.office_codes || %w[1A123B 2A555X]
```

Comment:
Assuming this fallback if no office codes exists was in there for testing only and that it should be removed

Reply:
(If yes, can we put TODOs on everything that will need removing)

Reply:
Might as well be now yeah